Snapshots/Snapshots.cpp

```diff
@@ -831,7 +840,10 @@ void SWS_SnapshotsWnd::GetOptions()
 	if (IsDlgButtonChecked(m_hwnd, IDC_MIX) == BST_CHECKED)
 	{
 		m_iSelType = 0;
-		g_iMask = MIX_MASK;
+		if (atof(GetAppVersion()) < 6)
+			g_iMask = MIX_MASK_NO_OFFS;
+		else
+			g_iMask = MIX_MASK;
```

Comment:
How about something like this here and in `SetSnapType`? (In the case of new future features having to be disabled according to the running REAPER version, having the check(s) in one place should be slightly easier to maintain.)

```cpp
g_iMask = MIX_MASK & g_compatMask;
```

Where `static int g_compatMask = ALL_MASK`, and in `SnapshotsInit`:
```cpp
// disable features unavailable in the running version of REAPER
if (atof(GetAppVersion()) < 6)
  g_compatMask &= ~PLAY_OFFSET_MASK;
```

Reply:
I like the elegantness of this. Thanks.